routes/index.js

```diff
@@ -9,38 +11,43 @@ function appendLocals(req, res) {
 
     res.locals.fullUrl = proto + '://' + req.hostname + req.path;
 
+    res.locals.theme = req.query.theme;
+
+    res.locals.displayTitle = function(subtitle) {
+        return TITLE + ': ' + subtitle + ' by MaxCDN';
```

Comment:
So, it would show up as `BootstrapCDN: Quick Start by MaxCDN`?

Not sure it's ideal. Maybe `Quick Start &mdash; BootstrapCDN by MaxCDN` is better. (replace mdash with the Unicode symbol if possible.